tests/introspection/tests.py

```diff
@@ -268,9 +268,15 @@ def assertDetails(details, cols, primary_key=False, unique=False, index=False, c
             elif details['columns'] == ['up_votes'] and details['check']:
                 assertDetails(details, ['up_votes'], check=True)
                 field_constraints.add(name)
+            elif details['columns'] == ['voting_number'] and details['check']:
+                assertDetails(details, ['voting_number'], check=True)
+                field_constraints.add(name)
```

Comment:
We can also check the name:
`self.assertTrue(name.startswith('__unnamed_constraint'))`

Reply:
I don't think it is necessary, we used a similar solution on SQLite.